BUILDING-configure.md

```diff
@@ -1,29 +1,64 @@
 Building using `configure`
 ==========================
 
-The build requires `libpq`, the C client library for PostgreSQL.  This library
-must be installed before you can build libpqxx.  You'll need the headers as
-well as the library binary.
-
 The instructions will assume that you're working from a command-line shell.
 If you prefer to work from an IDE, you'll have to know how your IDE likes to
 do things, and you'll want to follow the shell instructions as a guide.
 
+Prerequisites
+-------------
+
+Before you can begin the build process for libpqxx, you must first install
+`libpq-dev` and the latest `libpq5` library. These are C client libraries
+for PostgreSQL. 
+
+Enter this terminal command to obtain the latest `libpq5` library.
+
+```shell
+    apt-cache policy libpq5
+```
+
+From the output, select the `Candidate` value.
+
+For example, on Linux (Ubuntu 22.04) the terminal output is as follows:
+
+```shell
+libpq5:
+  Installed: 14.7-0ubuntu0.22.04.1
+  Candidate: 14.7-0ubuntu0.22.04.1
+  Version table:
+ *** 14.7-0ubuntu0.22.04.1 500
+        500 http://apt.pop-os.org/ubuntu jammy-security/main amd64 Packages
+        500 http://apt.pop-os.org/ubuntu jammy-updates/main amd64 Packages
+        100 /var/lib/dpkg/status
+     14.2-1ubuntu1 500
+        500 http://apt.pop-os.org/ubuntu jammy/main amd64 Packages
+```
+
+In the above output, the `Candidate` value is `14.7-0ubuntu0.22.04.1`. 
+Using this information, construct the library install command.
+
+```shell
+    sudo apt-get install libpq5=14.7-0ubuntu0.22.04.1 && sudo apt-get install libpq-dev
+```
+
+When the installation process completes, you are prepared to install libpqxx.
 
 Quick start
 -----------
 
-If you just want to get it built and installed quickly, try:
+If you just want to get it built and installed quickly, try the following:
 
 ```shell
-    ./configure
+    cd ~/Downloads
+    git clone https://github.com/jtv/libpqxx.git && cd libpqxx
```

Comment:
It's a fair thing to want to add, but there's a bit of a chicken-and-egg situation: people reading this file will often already have downloaded the source.

(Also, `~/Downloads` again is a bit specific, so I'd rather  leave that out.)

Reply:
From competing information around the web, and from the ways I've installed libraries like this on different projects, I started by downloading this to one of the /usr folders in the root dir. It was a total mess. Added many hours to undo the damage. 

I can't remember exactly how I came to go down the wrong path. Part of it, again, was that on a different project we did some type of install in /usr/bin and sometimes in /opt

There wasn't anything listed on libpqxx about how specifically it would install. I think I was also confused about whether I was supposed to put this in the directory of my project, and at some point while I was trying to get libpqxx to work, I think I tried also copy/pasting the libraries into the, /include dir of my project to see if that would work.

It's your call. I'm not necessarily a new programmer, but I'm not either highly experienced. A second year student in a cs program and three year's professional experience as a tech doc writer and junior-level web programmer. The linking process is not too familiar to me, but I'm learning. I don't know how well that translates to the rest of your audience, just sharing for whatever it's worth.

Reply:
Perhaps we can say something like "put this in a disposable directory"?  And then separately from that, we can add a section on where to install, and how to make use of what you've installed.

Reply:
I just noticed your response here.

Let me think on it and I'll get back to you.

Reply:
Did any solutions suggest themselves, @blue-santa?  Would like to get this merged.  Doesn't have to be perfect the first time around, so we could go for a quick merge and patch it up later.  And I don't want to make this unnecessarily slow and painful for you.

Reply:
I agree that it is probably best to merge as is and not worry about the `libpq5=9` issue. 

If you hear of someone else having this issue, though, ping me. I can try to help. 

The package content management stuff does seem to be a bit out of my experience level. I would need a dev familiar with how the system works to poke around with me and help me figure it out.

In the meantime, I did a double check just now and the latest iteration seems to avoid many of the more complicated issues.

We should be good to merge.

Reply:
Excellent.  Thanks again for this, and sorry for making it so difficult to contribute.  I can be a bit nitpicky.